build/env.py

```diff
@@ -0,0 +1,98 @@
+import os
+import shutil
+import subprocess
+import sys
+import sysconfig
+import tempfile
+import types
+
+
+if False:  # TYPE_CHECKING  # pragma: no cover
+    from typing import Dict, Optional, Iterable, Type
+
+
+class IsolatedEnvironment(object):
+    '''
```

Comment:
Is this `'''` style already consistent across the whole package? It's very unusual to use single-quotes for docstrings, even among code-bases that exclusively use single-quotes for strings (I even have them highlighted differently from `'''` strings in my syntax highlighter).

Reply:
+1 on this

Reply:
It is. I am open to change it, but it should come in a separate PR.